tests/python/frontend/onnx/test_forward.py

```diff
@@ -7933,6 +7933,77 @@ def verify_linear_regressor(a_shape, c_shape, i_shape, targets=1, batch=1):
     verify_linear_regressor((1, 4), (3), (1))
 
 
+@tvm.testing.parametrize_targets
+def test_dft(target, dev):
+    """test_dft"""
+
+    def verify_dft(
+        _axis,
+        _inverse,
+        _onesided,
+        _input,
+        _dft_length=None,
+    ):
+        input_names = ["input"]
+        if _dft_length is not None:
+            input_names.append("dft_length")
+
+        node = onnx.helper.make_node(
+            "DFT",
+            inputs=input_names,
+            outputs=["output"],
+            axis=_axis,
+            inverse=_inverse,
+            onesided=_onesided,
+        )
+
+        nodes = []
+        if _dft_length is not None:
+            nodes.append(
+                make_constant_node("dft_length", TensorProto.INT32, [], [_dft_length]),
+            )
+        nodes.append(node)
+
+        graph = helper.make_graph(
+            nodes,
+            "dft_test",
+            inputs=[
+                helper.make_tensor_value_info("input", TensorProto.FLOAT, input_shape),
+            ],
+            outputs=[
+                helper.make_tensor_value_info("output", TensorProto.FLOAT, output_shape),
+            ],
+        )
+
+        model = helper.make_model(graph, producer_name="dft_test")
+
+        verify_with_ort_with_inputs(
+            model,
+            [_input],
+            [input_shape],
+            target=target,
+            dev=dev,
+            rtol=1e-4,
+            atol=1e-4,
+            use_vm=False,
+        )
+
+    batch_size = 5
+    n = 2
+    D = 7
+
+    for axis in list(range(1, n)) + [-2]:
+        for inverse, onesided in [(0, 0), (0, 1), (1, 0)]:
+            for n_fft in [D, D - 1, D + 1]:
+                for c in [1, 2]:
+                    input_shape = [batch_size] + n * [D] + [c]
+                    output_shape = [batch_size] + n * [D] + [2]
+                    if onesided == 1:
+                        output_shape[axis] = output_shape[axis] // 2 + 1
```

Comment:
`output_shape` is not used

Reply:
This is used above in `helper.make_graph`

Reply:
Then just pass `input_shape` and `output_shape` to `verify_dft`. This is not a good programming practice.

Reply:
done